samples/subsys/debug/tracing/src/main.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <sys/util.h>
+#include <string.h>
+
+void main(void)
+{
+	u8_t count = 0;
+
+	if (strlen(CONFIG_UART_CONSOLE_ON_DEV_NAME) !=
+	    strlen("CDC_ACM_0") ||
+	    strncmp(CONFIG_UART_CONSOLE_ON_DEV_NAME, "CDC_ACM_0",
+		    strlen(CONFIG_UART_CONSOLE_ON_DEV_NAME))) {
+		printk("Error: Console device name is not USB ACM\n");
+
+		return;
+	}
```

Comment:
Given that 1) the while-loop below is the demo to be traced and 2) the location of this main.c -- then it would be nice if this demo app wasn't specific for any particular tracing backend.
This code might as well be part of the compiled-in backend's init(), no?

Reply:
@mped-oticon updated. Thanks